dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -274,23 +275,50 @@ CWPKI2059I=CWPKI2059I: Certificate revocation status checking found that the cer
+CWPKI2060E=CWPKI2060E: The OCSP URL from the certificate with the {0} serial number was not retrieved. The error is: ''{1}''.
 CWPKI2060E.explanation=The OCSP URL was not retrieved.
 CWPKI2060E.useraction=Ensure that the certificate is a valid X.509 certificate. If it is not valid, request a new certificate.
 
-CWPKI2061E=CWPKI2061E: The CRL distribution points from the certificate with the {0} serial number weren''t retrieved. The error is ''{1}''.
+CWPKI2061E=CWPKI2061E: The CRL distribution points from the certificate with the {0} serial number were not retrieved. The error is ''{1}''.
 CWPKI2061E.explanation=The CRL distribution points were not retrieved.
 CWPKI2061E.useraction=Ensure that the certificate is a valid X.509 certificate. If it is not valid, request a new certificate.
 
 CWPKI2062E=CWPKI2062E: The {0} OCSP responder URL defined in the server configuration is not a valid URI. If defined, it must be a valid URI to override the OSCP responder URL contained in the certificate.
 CWPKI2062E.explanation=Certificate revocation checking requires a valid OCSP responder URL.
 CWPKI2062E.useraction=Provide a valid OCSP responder URL in the server configuration.
 
-
 CWPKI2063E=CWPKI2063E: The ACME certificate authority at the {0} URI updated its terms of service and now requires the user to agree to the new terms of service at the following URI before it processes any further requests: {1}
 CWPKI2063E.explanation=The certificate authority updated its terms of service and requires user interaction.
 CWPKI2063E.useraction=Review the provided terms of service.
 
 CWPKI2064I=CWPKI2064I: The ACME service retrieved the certificate with the {0} serial number from the {1} URI in {2} seconds.
 CWPKI2064I.explanation=The ACME service successfully requested a certificate.
-CWPKI2064I.useraction=No action is required.
\ No newline at end of file
+CWPKI2064I.useraction=No action is required.
+
+CWPKI2065W=CWPKI2065W: The ACME service failed to automatically renew the certificate with the {0} serial number. The request is scheduled to try again in {1}. The certificate expires on {2}. The renew request error is ''{3}''.
+CWPKI2065W.explanation=The ACME service tried to renew a certificate but encountered an error. The ACME service continues to request a new certificate until a new certificate is issued.
+CWPKI2065W.useraction=Review the error message for details on the failure.
+
+CWPKI2066E=CWPKI2066E: The ACME service failed to automatically renew the certificate with the {0} serial number. The certificate is revoked. The request is scheduled to try again in {1}. The renew request error is ''{2}''.
+CWPKI2066E.explanation=The ACME service tried to renew a certificate but encountered an error. The ACME service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.
```

Comment:
Change  `SSL/TLS requests` to  `SSL and TLS requests `

Reply:
Changed